fixtures/c83be334b785044137611770c677d106

```diff
@@ -0,0 +1,8286 @@
+{
```

Comment:
It might be better to put this file in separate folder like `fixtures/cache` otherwise it's not clear how file with such a strange name is used

Reply:
Fixed!